scripts/helpers/multiedit.py

```diff
@@ -25,38 +25,54 @@
 
 else:
     #
-    # Now build up arguments
+    # Determine the changes the user has requested. Convert each change to
+    # its backend format.
     #
 
     fields = args['fields'].split(',')
     changes = {}
 
+    is_get_changes_success = True
     for f in fields:
-        # The value should be really filterEdited. This logic is missing and
-        # should be added to fix changing the inability to save Component
-        # on the multiedit page.
-        changes[allFields[f].savename] = args[f]
+        if not is_get_changes_success:
+            continue
 
-    try:
-        updateBug(args['Identifier'], changes)
+        try:
+            changes[allFields[f].savename] = allFields[f].filterEdit(args, args[f])
+        except Exception as err:
+            # There's an issue with the conversion to the backend format.
+            is_get_changes_success = False
+            queueMessage(MSG_ERROR,
+                         "Failed to save changes to {}. Unable to convert {} value '{}': {}".format(
+                             args['Identifier'], f, args[f], str(err)))
+            printXMLPageHeader(extraheaders=['Status: 500'])
+            printXMLMessages()
+            printXMLPageFooter()
 
-        data = loadBugs([args['Identifier']])[0]
+    if is_get_changes_success:
```

Comment:
Infact you *could* do away with this variable entirely using the for-else python syntax where the else block only gets run if you don't "break" from the loop.
Not sure if that's actually nicer though